File: 2021/July/Day 16/twoSum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        std::vector<int>::iterator it;
        vector<int> out;
        for(int i=0;i<size(nums);i++)
        {
            it = std::find(nums.begin()+i+1,nums.end(),target-nums[i]);

            if(it!= nums.end())
            {

                out.insert(out.end(),i);
                out.insert(out.end(),it - nums.begin());
                return out;
            }
            cout<<i;

        }
        return out;
    }
};

File: 2021/November/Day 29/twoSum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {

        int [] v = new int[]{};
        for(int i=0;i<nums.Length;i++)
        {
            for(int j=i+1;j<nums.Length;j++)
            {
                if(nums[j]==target-nums[i])
                {
                    v = v.Append(i).ToArray();
                     v = v.Append(j).ToArray();
                    return v;
                }
            }
        }
        return v;
    }
}
